userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -292,6 +274,36 @@ of the following:
 [wordmark]: https://en.wikipedia.org/wiki/Wordmark
 [your logo]: /docs/adding-content/iconsimages/#add-your-logo
 
+### Light/Dark mode menu
+
+To enable the display of a Light-/[Dark-mode] menu in the navbar, set
```

Comment:
Hmm, which do you prefer @LisaFC: "Light/Dark mode menu" or "Light-/Dark-mode menu" or something else?

Reply:
Whatever we choose, it should be consistent with the heading, which it isn't right now.
I would go for `Light/Dark mode`.